pkg/lib/record_continuation_test.go

```diff
@@ -1,45 +0,0 @@
-// Copyright 2022 The Moov Authors
```

Comment:
This test covered an unused component. I removed the component, and therefore this test as well.